osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSlider.cs

```diff
@@ -176,8 +177,6 @@ protected override void OnFree()
         {
             base.OnFree();
             trackingCheckpoints.Clear();
-
-            slidingSample.Samples = null;
```

Comment:
Would be better if we did `slidingSample?.ClearSamples()`. We don't want samples to fall over into other sliders whenever we're pooling.